<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>771</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>454</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_file">
     <attribute name="title">
      <string>File</string>
     </attribute>
     <widget class="QPushButton" name="bt_open">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_save">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>save as</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_open">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>561</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lena256.bmp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_save">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>561</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>new-Lena256.bmp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_img_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_set_file_root">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>set root</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_file_root">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>561</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>F:\dataset\ex_images\</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_pixel">
     <attribute name="title">
      <string>Pixel</string>
     </attribute>
     <widget class="QPushButton" name="bt_set_pixel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>set pixel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_x">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_y">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_b">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_g">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_r">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="et_gray">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gray</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_get_pixel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>104</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>get pixel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_show_color">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string> Pixel Color</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_img_test">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionsdf"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsdf">
   <property name="text">
    <string>(1)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>(2)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
